types/rox-react-native/index.d.ts

```diff
@@ -40,25 +36,26 @@ export function setup(
 
 export interface RoxSetupOptions {
     version?: string;
-    // https://support.rollout.io/docs/javascript-browser-api#section-configurationfetchedhandler
+    // https://support.rollout.io/docs/reactnative#section-configurationfetchedhandler
     configurationFetchedHandler?(fetcherResult: RoxFetcherResult): void;
     debugLevel?: 'verbose';
-    // https://support.rollout.io/docs/javascript-browser-api#section-using-the-impressionhandler-option
-    impressionHandler?(
-        reporting: RoxReporting,
-        experiment: RoxExperiment,
-        context: unknown
-    ): void;
+    // https://support.rollout.io/docs/reactnative#section-using-the-impressionhandler-option
+    impressionHandler?(reporting: RoxReporting, experiment: RoxExperiment, context: unknown): void;
     platform?: string;
-    freeze?: 'untilLaunch' | 'none';
+    freeze?: FreezeOptions;
+    disableNetworkFetch?: boolean;
+    devModeSecret?: string;
```

Comment:
Can you add the `AsyncStorage` option as well 🙏 
https://support.rollout.io/docs/reactnative#section--asyncstorage-

Reply:
Done, with type 'any' though.

Reply:
Thanks 🙏